Tests/IntegrationTests/Components Tests/PayTo/PayToComponentTests.swift

```diff
@@ -20,21 +30,11 @@ class PayToComponentTests: XCTestCase {
     }
 
     func test_paymentMethodType_isPayto() throws {
-        let sut = try PayToComponent(
```

Comment:
I see a risk of tests not being atomic anymore that some time when the logic becomes convoluted and amount of tests increase we will have side effects applied to sut based on tests execution order. I suggest creating `sut` every time by using `makeSUT()` method that incapsulates all possible variations via parameters injection.